src/System.IO.MemoryMappedFiles/tests/MemoryMappedViewStream.Tests.cs

```diff
@@ -165,6 +166,24 @@ public void PointerOffsetMatchesViewStart()
             }
         }
 
+        [Fact]
+        public void NonAlignedOffsets()
+        {
+            var bytes = Enumerable.Range(1, 70000).Select(i => unchecked((byte)i)).ToArray();
+            var file = GetTestFilePath();
+            File.WriteAllBytes(file, bytes);
+            using (var mmf = MemoryMappedFile.CreateFromFile(file))
+            {
+                using (var stream = mmf.CreateViewStream(68000, 10))
```

Comment:
This test does not. I added it to validate that my change doesn't break things. We actually don't have any tests that fail without these changes. I will add one for that case.